### ISSUE TITLE
[Request] Add proper dark mode support.

### ISSUE THREAD

**Issue by Theguysayhi** [nimble0/dotterel#25]
I think I speak for everyone when I say that the white background is rather unsightly. An option to have it black would be great!

Fantastic work on the app. It's inspired me to learn how to use such an amazing keyboard.

**Comment by nimble0** [comment]
Good idea, it should follow the system dark mode setting in the latest release.
